chain/network/src/peer.rs

```diff
@@ -199,6 +199,9 @@ impl Peer {
             PeerMessage::BlockRequest(h) => self.tracker.push_request(*h),
             _ => (),
         };
+
+        trace!(target: "diagnostic", key="tx", msg=%ser(&msg));
```

Comment:
@ailisp This is the new way to make diagnostic using tracing! Not all advantages of tracing are leveraged here, but I'm working on it. They have a very cool concept (spans) that integrate very nice with async programs.

Eventually we should migrate all logging to structured logging for the sake of simplicity, but fortunately using tracing this process can be done on demand.

Reply:
Cool, this looks neat